tests/E2E/test/campaigns/full/02_product/10_check_product_page_buttons.js

```diff
@@ -34,7 +34,14 @@ scenario('Check product page buttons', () => {
         .then(() => client.waitForExistAndClick(AddProductPage.preview_buttons));
     });
     test('should switch to the Front Office', () => client.switchWindow(1));
-    test('should click on the "Preview" link', () => client.waitForExistAndClick(AddProductPage.preview_link));
+    test('should click on the "Preview" link', () => {
+      return promise
+        .then(() => {
+          if (global.ps_mode_dev) {
+            client.waitForExistAndClick(AddProductPage.preview_link)
+          }
+        })
+    });
```

Comment:
I see this code written two times. You should move this code in a common secnario file, and call it from here.